puppet/modules/chassis/manifests/init.pp

```diff
@@ -23,6 +23,10 @@
 		require    => Package['nginx']
 	}
 
+	file {['/etc/nginx', '/etc/nginx/sites-available', '/etc/nginx/sites-enabled']:
```

Comment:
I think we need to work out some coding standards for our Puppet files. I kinda want to see more spaces inside '{' and '}'. That's not a blocker for this just something to note so we can work on that going forward.